### PR TITLE
fix dom-highlights rendering under absolute position elements

### LINKED ISSUE #7762: Element highlights not visible on certain position:absolute elements
Element dom highlights can render underneath `position:absolute` and `fixed` elements due to the outer highlight container not having `position: absolute` and a high `z-index`

### PR DESCRIPTION
**Pull request by kuceb**
- fix dom highlights rendering below certain position `absolute` elements

- fix #7762 

### User facing changelog
Bug fix: fixed issue causing element highlights to be invisible when targeting absolute positioned elements

### How has the user experience changed?
before: dom highlights rendering underneath `position:absolute` element:
image

after: correct dom highlight:
image

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue been tagged with a release in ZenHub?

### CODE REVIEW

File: packages/driver/cypress/integration/e2e/dom_hitbox.spec.js [hunk 1]
```diff
@@ -61,6 +62,23 @@ describe('rect highlight', () => {
     ensureCorrectHighlightPositions('#button')
     ensureCorrectTargetPosition('#button')
   })
+
```
Comment (jennifer-shehane) [inline packages/driver/cypress/integration/e2e/dom_hitbox.spec.js:65]:
```suggestion

// https://github.com/cypress-io/cypress/issues/7762
```

**Review by jennifer-shehane — DISMISSED** [review]
Tested manually. Looks good. Glad to see tests for CSS changes. 😄

**Review by jennifer-shehane — DISMISSED** [review]

**Review by jennifer-shehane — APPROVED** [review]

File: packages/runner/src/lib/dom.js [hunk 2]
```diff
@@ -73,6 +73,8 @@ function addElementBoxModelLayers ($el, $body) {
   const $container = $('<div class="__cypress-highlight">')
   .css({
     opacity: 0.7,
+    position: 'absolute',
+    zIndex: 2147483647,
```
Comment (chrisbreiding) [inline packages/runner/src/lib/dom.js:77]: My favorite z-index value 😆

**Review by chrisbreiding — APPROVED** [review]
Looks good 👍

**Pull request merged**